cadence/contracts/bridge/FlowEVMBridgeUtils.cdc

```diff
@@ -700,8 +700,11 @@ contract FlowEVMBridgeUtils {
                 "bridged": "EVMVMBridgedToken"
             }
         }
-        let bridgeFactoryEVMAddressBytes: [UInt8] = bridgeFactoryEVMAddress.decodeHex()
-        self.bridgeFactoryEVMAddress = self.getEVMAddressFromHexString(address: bridgeFactoryEVMAddress)
-            ?? panic("Invalid bridge factory EVM address")
+        // Deploy the FlowBridgeFactory.sol contract from provided bytecode and capture the deployed address
+        self.bridgeFactoryEVMAddress = self.borrowCOA().deploy(
```

Comment:
Something to note: The return value of `deploy` is going to change. See: https://github.com/onflow/flow-go/pull/5606. It will return an `EVM.Result`, where the data is actually the deployed contract address. Maybe we should add a TODO, or an issue to keep track of it.

Reply:
Thanks for the heads up! I'll make an issue